spaceship/spec/portal/fixtures/getProvisioningProfileAppStore.action.json

```diff
@@ -84,7 +84,6 @@
       "hasAskKey": false
     }],
     "devices": [],
-    "deviceIds": ["FVRY7XH22J", "4ZE252U553"],
```

Comment:
This app store profile shouldn't have had devices in it. The fixture data was wrong.